runtime/near-vm-runner/src/tests.rs

```diff
@@ -134,3 +133,14 @@ fn gas_and_error_match(
     assert_eq!(outcome.burnt_gas, expected_gas, "burnt gas differs");
     assert_eq!(outcome_and_error.1, expected_error);
 }
+
+/// Small helper to compute expected loading gas cost charged before loading.
+/// Includes some hard-coded parameter values that would have to be updated
+/// if they change in the future.
```

Comment:
Can we add a brief comment as to how these hardcoded values were derived please?  To help newbies like myself 😄

Reply:
Yes, I elaborated it some more. But the key here is that `parameter` is a technical term. It is a runtime parameter that is part of protocol configuration.